virus/sedimentum_occultus.md

```diff
@@ -0,0 +1,111 @@
+
+### Virus Genome
+
+Still Image:
+
+![](img/sedimentum_occultus/figure_1.png)
+
+[Interactive Version](https://www.ncbi.nlm.nih.gov/projects/sviewer/?id=lcl|1&tracks=[key:sequence_track,name:Sequence,display_name:Sequence,id:STD649220238,category:Sequence,annots:Sequence,ShowLabel:false,ColorGaps:false,shown:true,order:0][key:gene_model_track,name:U2EZpwA2z898C,display_name:(U)%20ORFfinder_12.4.72257490,id:U2EZpwA2z898C,data_key:dcrjGxXIwunu5vTuxffK4Jmdm5yTnr-Ut4y5piyrKgnqPwqyJwYWXVoPZHYxamhyelotW25FflRkTnRRQnx7fltOfQ,dbname:NetCache,category:ORFfinder,annots:ORFfinder_12.4.72257490_UUD1701692577DUU_gene_model,Options:MergeAll,CDSProductFeats:false,NtRuler:true,AaRuler:true,HighlightMode:2,ShowLabel:true,highlights_color:#0000ff,shown:true,order:1]&key=_kFokJ5DSWJlbX9lTnxBaxIWEBcYFTQfPAcyLacgoYJitBa9sNaB6w-kRN0RwUjZWvEN8E7uXv9E5VT6Ytdb1XvlXQ&mk=3479:3502|C|red,3797:3832|A|0000FF,3584:3625|B|00FF00,2969:8341|ORF1|black|9,247:2679|ORF2|333333|1,3050:4282|RdRp|FF00FF&noguess_assm=1&spacing=1&v=1:8971&c=null&gflip=1&select=null&slim=0&toolbar=-d) (may not work)
+
+**Figure 1:** Genome of *Sedimentum occultus*. The ORF1 region (in black) shows the location of ORF1, which contains the nucleotide sequence for RdRp. The RdRp region (in pink) shows the location of the RdRp nucleotide sequence that is directly translated into RdRp. Within this region are the motifs A, B, and C (in blue, green, and red, respectively). The ORF2 region (in black) shows the location of ORF2, which contains nucleotide sequences for capsid proteins.
+
+### RdRp Structure
+
+![](img/sedimentum_occultus/figure_2.png)
+
+**Figure 2**: Structure of the RdRp as determined by AlphaFold2. The A, B, and C motifs are shown in blue, green, and red, respectively.
+
+### RdRp Multiple Sequence Alignment
+
+Still Image:
+
+![](img/sedimentum_occultus/figure_3.png)
+
+[Interactive Version](https://www.ncbi.nlm.nih.gov/projects/msaviewer/?anchor=0&coloring=rasmol&key=E6yFfXOupI-IgJKIo5Gshv_7_fr1-Nny0erfwErNTGRdUi6QH18umCqwvcno1bHNo-X05Lf6p-u98a3um8OiwYLxpA,HaKLc32gqoGGjpyGrZ-iiPH18_T79tf83-TRzkTDQmpTXFnzBsA3zriYm-HO_Zflhc3SzJHSgcOb2YvGveuE6aTZgg&track_config=protein_default&hidden=4,6,7,8,9,10,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100&columns=d:120,b:55,x:17,aln,e:55,o:150) (may not work)
+
+**Figure 3:** Multiple sequence alignment of the *Sedimentum occultus* RdRp protein sequence to all BLAST hits with >80% query coverage and >40% amino acid identity.
```

Comment:
Could you include a `.fa` of your multiple sequence alignment in the data folder please.